### PR TITLE
#435 Fixed actions and views

### PR DESCRIPTION
**Pull request by jpyzio**
After resolve action problems, there was problems in views, crud generete for example:

In client/index.html.twig:
```
<td>{% if entity.isIndividual %}Yes{% else %}No{% endif %}</td>
```
insted:
```
<td>{% if client.isIndividual %}Yes{% else %}No{% endif %}</td>
```

It was fixed too in commit.

### CODE REVIEW

File: Resources/skeleton/crud/actions/edit.php.twig [hunk 1]
```diff
@@ -17,7 +17,7 @@
     {
 {% block method_body %}
         $deleteForm = $this->createDeleteForm(${{ entity_singularized }});
-        $editForm = $this->createForm(new {{ entity_class }}Type(), ${{ entity_singularized }});
+        $editForm = $this->createForm({{ entity_class }}Type::class, ${{ entity_singularized }});
```
Comment (xabbuh) [inline Resources/skeleton/crud/actions/edit.php.twig]: Actually, this does not work in PHP versions before 5.5. To be on the safe side you should probably rather use the FCQN instead.

Comment (OskarStark) [inline Resources/skeleton/crud/actions/edit.php.twig]: :+1: for FQN

Comment (OskarStark) [inline Resources/skeleton/crud/actions/edit.php.twig]: maybe a check if php > 5.3 is installed or something like this or an option in the commands?

File: Command/GenerateCommandCommand.php [hunk 2]
```diff
@@ -43,7 +43,7 @@ public function configure()
 inside bundles. Provide the bundle name as the first argument and the command
 name as the second argument:
 
-<info>php app/console generate:command AppBundle blog:publish-posts</info>
+<info>php bin/console generate:command AppBundle blog:publish-posts</info>
```
Comment (fabpot) [inline Command/GenerateCommandCommand.php]: All the `app/` to `bin/` changes must be reverted as we support both 2.x and 3.x dir structure.

Comment (jpyzio) [inline Command/GenerateCommandCommand.php]: Done.

**Comment by khalid-s** [comment]
hi, just gave a try for this solution, and works great.

But when generating a crud for a bundle living in a vendor name, there is a \ missing between the vendor name and the bundle name for new and edit action. not sure that am clear. e.g

For a bundle CompanyBundle in the folder  /src/IKNSA/CompanyBundle/Controller/CompanyController.php 

```
public function newAction(Request $request)
    {
        $company = new Company();
        $form = $this->createForm('\IKNSACompanyBundle\Form\CompanyType', $company);
        $form->handleRequest($request);
```
There is a backslash missing between IKNSA and CompanyBundle

File: Resources/skeleton/crud/actions/edit.php.twig [hunk 3]
```diff
@@ -17,7 +17,7 @@
     {
 {% block method_body %}
         $deleteForm = $this->createDeleteForm(${{ entity_singularized }});
-        $editForm = $this->createForm(new {{ entity_class }}Type(), ${{ entity_singularized }});
+        $editForm = $this->createForm('\{{ bundle }}\Form\{{ entity_class }}Type', ${{ entity_singularized }});
```
Comment (fabpot) [inline Resources/skeleton/crud/actions/edit.php.twig]: The leading `\` is not needed, same below.

Comment (fabpot) [inline Resources/skeleton/crud/actions/edit.php.twig]: IIRC, `bundle` references the bundle name, not the bundle namespace.

**Comment by jpyzio** [comment]
It should work fine.

**Comment by khalid-s** [comment]
Tested again and it works fine

**Comment by fabpot** [comment]
Thank you @jpyzio.

**Pull request closed without merging**